### PR TITLE
Get the local utc offset, and pass it to alarms

### PR DESCRIPTION
**Pull request by MrZammler**
##### Summary

Get the local utc offset from strftime, and pass it to alarms. This is for the cloud to produce a local time for the email notification from utc time.

##### Component Name

Health

### CODE REVIEW

File: daemon/main.c [hunk 1]
```diff
@@ -693,6 +693,25 @@ static void get_system_timezone(void) {
...
+        char helper[6];
+
+        t = now_realtime_sec();
+        tmp = localtime_r(&t, &tmbuf);
+
+        if (tmp != NULL) {
+            if(strftime(helper, 6, "%z", tmp) == 0)
+                buffer[0] = '\0';
+
+            netdata_utc_offset = strdupz(helper);
```
Comment (stelfrag) [inline daemon/main.c:712]: This is undefined on strftime failure

Comment (MrZammler) [reply daemon/main.c]: I've changed it with https://github.com/netdata/netdata/pull/10899/commits/e8638fe0cfebe1a3509447e9588b5072f0980f7c and added +0000 if it fails....

File: daemon/main.c [hunk 2]
```diff
@@ -693,6 +693,28 @@ static void get_system_timezone(void) {
...
+        tmp = localtime_r(&t, &tmbuf);
+
+        if (tmp != NULL) {
+            if(strftime(helper, 6, "%z", tmp) == 0)
+                netdata_utc_offset = strdupz("+0000");
+            else
+                netdata_utc_offset = strdupz(helper);
+        }
+
+        info ("[%ld]", tmbuf.tm_gmtoff);
```
Comment (thiagoftsm) [inline daemon/main.c:715]: please, add more information in this info message to simplify the identification of the value. Netdata stores a lot of information inside the `error.log`.

Comment (MrZammler) [reply daemon/main.c:715]: Hi Thiago, sorry for the mess... this is changing at this moment, so that piece of code will go away.

**Pull request merged**